src/modules/previewpane/MonacoPreviewHandler/MonacoPreviewHandlerControl.cs

```diff
@@ -91,11 +91,11 @@ public override void DoPreview<T>(T dataSource)
 
                                     _webView2Environment = webView2EnvironmentAwaiter.GetResult();
                                     var vsCodeLangSet = FileHandler.GetLanguage(Path.GetExtension(filePath));
-                                    var fileContent = File.ReadAllText(filePath);
+                                    var fileContent = new StreamReader(new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite)).ReadToEnd();
                                     var base64FileCode = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(fileContent));
 
                                     // prepping index html to load in
-                                    var html = File.ReadAllText(Settings.AssemblyDirectory + "\\index.html").Replace("\t", string.Empty, StringComparison.InvariantCulture);
+                                    var html = fileContent = new StreamReader(new FileStream(Settings.AssemblyDirectory + "\\index.html", FileMode.Open, FileAccess.Read, FileShare.ReadWrite)).ReadToEnd();
```

Comment:
any reason why are you overwriting `fileContent` here?

Reply:
Nope. My fault.